README.md

```diff
@@ -1,7 +1,31 @@
-# Vue example app for Nextcloud
+# Nextcloud Photos
+**📸 Your memories under your control**
 
-1. ☁ Clone this into your `apps` folder of your Nextcloud
-2. 👩‍💻 In a terminal, run the command `make dev-setup` to install the dependencies
-3. 🏗 Then to build the Javascript whenever you make changes, run `make build-js`
-4. ✅ Enable the app through the app management of your Nextcloud
-5. 🎉 Partytime!
+![](screenshots/screenshot1.png)
+
+## Why is this so awesome?
+- **📸 Beautiful Photo und Video Timeline**
+- **⭐ Favorites and Tagging:** Favorite and tag your photos!
+- **Slideshow and easy sharing** Show slideshows and share your photos or albums easy
+- **Albums:** Create albums from your contents
+    
+    Future plans:
+- ...
+
+## 🚀 Installation  
+In your Nextcloud, simply navigate to »Apps«, find the Photos app and enable it. Then open the app from the app menu. Consider installing the [preview generator](https://github.com/rullzer/previewgenerator) for pregenerating the thumnails. 
+Please have a look at [CHANGELOG.md](https://github.com/nextcloud/photos/blob/master/CHANGELOG.md) for details about changes.
+    
+## Mobile Photos
+Use the Android/iOS App to view your photos. It's possible to autoupload them.
+
+##Maintainers:
+- [John Molakvoæ](https://github.com/skjnldsv)
+If you'd like to join, just go through the [issue list](https://github.com/nextcloud/photots/issues) and fix some. :)
+    
+## 🏗 Development setup
```

Comment:
Would you mind adding an explanation for the linters? 
So everyone can lint and easily fix linting issues.
Also for vscode user we can add some tips for extensions.

Reply:
Do you mean adding the command "npm run lint:fix"?

Reply:
Yep